src/Microsoft.Security.Utilities.Core/PotentialSecurityKeys/SEC101_055_Pkcs12CertificatePrivateKeyBundle.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+#nullable enable
+
+namespace Microsoft.Security.Utilities
+{
+    public class Pkcs12CertificatePrivateKeyBundle : RegexPattern
+    {
+        // _truePositiveExamples is used during testing, including a stress test where this code absent caching
+        // is would case the stress test (n=1000) to take 10+ minutes to run due the expensive nature of generating
+        // private keys and certificates. We use Lazy here since during non-test execution we do not want to incur
+        // the cost of generating these examples.
+        private static Lazy<List<string>> _truePositiveExamples = new Lazy<List<string>>(() => {
```

Comment:
Nit for sure, but I'd prefer if this were next to GenerateTruePositiveExamples. I would like to get to the class constructor before the test cases.